react-native/android/src/main/jni/Android.mk

```diff
@@ -84,13 +84,6 @@ LOCAL_SRC_FILES += src/object-store/src/sync/impl/sync_metadata.cpp
 LOCAL_SRC_FILES += src/object-store/src/sync/impl/work_queue.cpp
 endif
 
-# Workaround for memmove/memcpy bug
-ifeq ($(strip $(TARGET_ARCH_ABI)),armeabi-v7a)
```

Comment:
What was the original reason this was only done for v7a?

Reply:
We have only observed it on ARM v7a. But for some reason, `Android.mk` is only applied once and not for each compilation. So the workaround is never enabled. If `memmove()` does not have the bug, the original `memmove()` is used - this is only an optimization (once in the app's life time) to avoid checking on all platforms. Removing this is under the "better safe than sorry" category.